Assets/__Scripts/UI/Options/AdbHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using QuestDumper;
+using UnityEngine;
+
+// This class is so stupid and I don't know why I made it so stupid
+// I'm sorry
+public class AdbHandler : MonoBehaviour
+{
+    private BetterToggle _betterToggle;
+
+    // I hate this
+    private bool toggledBySelf;
+    
+    private void Start()
+    {
+        _betterToggle = GetComponent<BetterToggle>();
+        // Set toggle
+
+        
+        // Update UI ugh
+        toggledBySelf = true;
+        _betterToggle.SetUiOn(Adb.IsAdbInstalled(out _));
+        toggledBySelf = false;
```

Comment:
If you want to clean this up, I'd throw out the `BetterToggle` entirely and use a regular `Toggle` (can use the CMUI prefab as a base), then you can remove this whole `toggledBySelf` bullshit and instead use `Toggle.SetValueWithoutNotify()`. Alternatively, you can just replicate that method in `BetterToggle`.